src/codegen/cppcg.cpp

```diff
@@ -1585,8 +1587,20 @@ void CppCodeGenerator::GenConstructor(PObjectBase class_obj, const EventVector &
 {
 	m_source->WriteLn();
 	m_source->WriteLn( GetCode( class_obj, wxT( "cons_def" ) ) );
-	m_source->WriteLn( wxT( "{" ) );
-	m_source->Indent();
+    if ( class_obj->GetPropertyAsInteger(wxT("create_children")) != 0 ) {
+        m_source->WriteLn( wxT( "{" ) );
+        m_source->Indent();
+        m_source->WriteLn( wxT( "Create();" ) );
+        m_source->Unindent();
+        m_source->WriteLn( wxT( "}" ) );
+    } else {
+        m_source->WriteLn( wxT( "{" ) );
+        m_source->WriteLn( wxT( "}" ) );
+    }
+    wxString className = class_obj->GetPropertyAsString( wxT( "name" ) );
+    m_source->WriteLn( wxString::Format( wxT( "void %s::Create()" ), className.c_str(), className.c_str() ) );
+    m_source->WriteLn( wxT( "{" ) );
+    m_source->Indent();
```

Comment:
This will break the subclass feature and this will only work for toplevel elements

Reply:
Not knowing any of the background to wxFormBuilder this was an attempt to make the generated code work for Android builds. Currently whilst the dialogs do build and display the result is quite often unreadable. If the name 'Create' is a problem that is easily changed. The default behaviour does not affect any current objects, the user will have to select to not auto create the children.

There is no current method of setting the default fonts to use other than with the 'SetFont' method after the dialog is instantiated. The current wxFormBuilder process does not allow for the default font to be set before the children are created. In Android the only way to then change the font is to cycle through all children and set their font explicitly.

I only use C++ so I only made the changes to affect the C++ part of the system. I do not know enough about the other languages to even start to do that.

If this solution is totally unacceptable then there needs to be a method of setting the default font before the children are created. I am not sure what you mean by 'code injection' or the use of a plugin, so I am in the dark about this possible solution.